cmd/nginx/flags.go

```diff
@@ -188,25 +192,47 @@ Feature backed by OpenResty Lua libraries.`)
 
 	parser.AnnotationsPrefix = *annotationsPrefix
 
-	// check port collisions
-	if !ing_net.IsPortAvailable(*httpPort) {
-		return false, nil, fmt.Errorf("Port %v is already in use. Please check the flag --http-port", *httpPort)
-	}
+	bindIpAddresses := make([]string, 0)
+	for _, ipAddressStr := range strings.Split(*bindAddresses, ",") {
+		ipAddressSanitized := strings.Replace(strings.Replace(ipAddressStr, "]", "", -1),
```

Comment:
The strings.Trim... functions are more suitable for this. Besides, do we need brackets at all? (cf. previous @aledbf's comment)